src/manageiq_client/api.py

```diff
@@ -438,6 +438,8 @@ def reload(self, expand=None, get=True, attributes=None):
                 self._data.update(new)
         self._actions = self._data.pop("actions", [])
         for key, value in self._data.items():
+            if value is None:
+                continue
```

Comment:
As you say, the question is should we just drop these, or do `setattr(self, key, None)` here.
If we drop them, we behave as we did before https://github.com/ManageIQ/manageiq-api/pull/253 change.

Long-term, it would be nice if all attributes were set, even if None, so you could access entity.attr without fear of exception, but I don't see a way this library can guarantee that.  Given manageiq omitted null fields until very recently, in most cases one needs defensive code (`hasattr` etc.) anyway where null is possible...

LGTM as is for now, just fixing the bug; we can always add the setattr later, if we decide it's helpful.